src/main/cpp/blaze_util_linux.cc

```diff
@@ -21,6 +21,9 @@
 #include <sys/statfs.h>
 #include <sys/types.h>
 #include <unistd.h>
+#include <stdio.h>
```

Comment:
That seemed to be the format used with the other includes in that file (e.g. signal.h instead of csignal, string.h instead of cstring).

I will admit the next line I added probably should not be, errno.h is already included here. Not sure why I added that.